src/service/url-replacements-impl.js

```diff
@@ -1043,93 +1030,8 @@ export class UrlReplacements {
-    if (isV2ExperimentOn) {
-      // TODO(ccordy) support opt_collectVars && opt_whitelist
-      return this.expander_./*OK*/expand(url, opt_bindings, opt_collectVars,
-          opt_sync, opt_whiteList);
-    }
-
-    // existing parsing method
-    const expr = this.variableSource_.getExpr(opt_bindings);
-    let replacementPromise;
-    let replacement = url.replace(expr, (match, name, opt_strargs) => {
-      let args = [];
-      if (typeof opt_strargs == 'string') {
-        args = opt_strargs.split(/,\s*/);
-      }
-      if (opt_whiteList && !opt_whiteList[name]) {
-        // Do not perform substitution and just return back the original
-        // match, so that the string doesn't change.
-        return match;
-      }
-      let binding;
-      if (opt_bindings && (name in opt_bindings)) {
-        binding = opt_bindings[name];
-      } else if ((binding = this.variableSource_.get(name))) {
-        if (opt_sync) {
-          binding = binding.sync;
-          if (!binding) {
-            user().error(TAG, 'ignoring async replacement key: ', name);
-            return '';
-          }
-        } else {
-          binding = binding.async || binding.sync;
-        }
-      }
-      let val;
-      try {
-        val = (typeof binding == 'function') ?
-          binding.apply(null, args) : binding;
-      } catch (e) {
-        // Report error, but do not disrupt URL replacement. This will
-        // interpolate as the empty string.
-        if (opt_sync) {
-          val = '';
-        }
-        rethrowAsync(e);
-      }
-      // In case the produced value is a promise, we don't actually
-      // replace anything here, but do it again when the promise resolves.
-      if (val && val.then) {
-        if (opt_sync) {
-          user().error(TAG, 'ignoring promise value for key: ', name);
-          return '';
-        }
-        /** @const {Promise<string>} */
-        const p = val.catch(err => {
-          // Report error, but do not disrupt URL replacement. This will
-          // interpolate as the empty string.
-          rethrowAsync(err);
-        }).then(v => {
-          replacement = replacement.replace(match,
-              NOENCODE_WHITELIST[match] ? v : encodeValue(v));
-          if (opt_collectVars) {
-            opt_collectVars[match] = v;
-          }
-        });
-        if (replacementPromise) {
-          replacementPromise = replacementPromise.then(() => p);
-        } else {
-          replacementPromise = p;
-        }
-        return match;
-      }
-      if (opt_collectVars) {
-        opt_collectVars[match] = val;
-      }
-      return NOENCODE_WHITELIST[match] ? val : encodeValue(val);
-    });
-
-    if (replacementPromise) {
-      replacementPromise = replacementPromise.then(() => replacement);
-    }
-
-    if (opt_sync) {
-      return replacement;
-    }
-    return replacementPromise || Promise.resolve(replacement);
+    return this.expander_./*OK*/expand(url, opt_bindings, opt_collectVars,
```

Comment:
Do we need to continue to alias here?

This method just returns the result of another method with the same arguments. Can the caller just call the underlying method?

Reply:
Good idea. Done.